elixir/apps/web/lib/web/components/navigation_components.ex

```diff
@@ -331,13 +331,14 @@ defmodule Web.NavigationComponents do
     <.website_link href={~p"/contact/sales"}>Contact Sales</.website_link>
   """
   attr :href, :string, required: true
+  attr :anchor, :string, required: false, default: ""
   slot :inner_block, required: true
   attr :rest, :global
 
   def website_link(assigns) do
     ~H"""
     <.link
-      navigate={"https://www.firezone.dev?utm_source=product#{@href}"}
+      navigate={"https://www.firezone.dev#{@href}?utm_source=product##{@anchor}"}
```

Comment:
Should we rename `href` to `path`?  An actual `href` can take a full URL, but in this case that would not work.

Reply:
Agreed, then if we change `anchor` to `fragment` it's consistent with the URL parts entirely https://en.wikipedia.org/wiki/URL#Syntax

Reply:
SGTM